### PR TITLE
added art's card

### PR DESCRIPTION
**Pull request by Artur-at-work**
added art-pak-card

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3474,7 +3493,7 @@ <h2>A project for first-time contributions</h2>
             <p class="name">Name</p>
             <p class="contact">
                 <i class="fa fa-twitter"></i>
-                <a href="YOUR twitter account link" target="_blank">@yourHandle</a>
+                <a href="YOUR twitter account link" target="_blank">@Artur-at-work</a>
```
Comment (Syknapse) [inline index.html]: This is the template card, and it should not be changed at all. You add your Twitter account only in your card. You will need to undo this so I can merge your card. Thanks.

**Comment by Syknapse** [comment]
Thanks for the PR, your card looks good, but there is one small change needed before I can merge it.

File: index.html (hunk 1 above)
Comment (Artur-at-work) [reply index.html]: Thank you for pointing that out. I have edited template to original, and pushed new commit.
Sorry for trouble.

Comment (Syknapse) [reply index.html]: no trouble at all. It's all part of the normal process. Your card has been merged 👍

**Pull request merged**